lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -265,6 +271,35 @@
     type: default_privs
     role: reader
 
+# Available since version 2.8
+# ALTER DEFAULT PRIVILEGES FOR ROLE librarian IN SCHEMA library GRANT SELECT ON TABLES TO reader
+# GRANT SELECT privileges for new TABLES objects created by librarian as
+# default to the role reader.
+# For specific
+- postgresql_privs:
```

Comment:
Examples should have a `name:` entry.
That is what we recommend to users, so we should set a good example in the docs.